LoopProjectFile/ExtractedInformation.py

```diff
@@ -302,26 +303,55 @@ def SetStratigraphicLog(root, data, append=False, verbose=False):
             zlib=True,
             complevel=9,
         )
+        
+        # Define compound type for thickness calculator (list of length 5)
+        thicknessCalculatorType_t = siGroup.createCompoundType(
+            LoopProjectFile.thicknessCalculatorType, "ThicknessCalculator"
+        )
+        siGroup.createVariable(
+            "thicknessCalculator",
+            thicknessCalculatorType_t,
+            zlib=True,
+            complevel=9,
+        )
     else:
         siGroup = resp["value"]
 
     if siGroup:
         stratigraphicLayersLocation = siGroup.variables["stratigraphicLayers"]
+        thickness_calculator = siGroup.variables["thicknessCalculator"]
```

Comment:
Is this being added to the projectFile? When i try to get the element using projectFile['thicknessCalculator'] it can't find the element.

Reply:
yep, I forgot to update the getter. Could you try now?
Minimum reproducible example working on my end:

```
import LoopProjectFile as LPF

resp = LPF.Get("local_source.loop3d", "stratigraphicLog")
if resp["errorFlag"]:
    print(resp["errorString"])
else:
    stratilog = resp["value"]

stratilog
```
new vars should be `thicknessCalculatorData` and `thicknessCalculatorActiveFlags`